packages/protocol/src/vaults/borrowing/BorrowingVault.sol

```diff
@@ -46,6 +46,8 @@ contract BorrowingVault is BaseVault {
   error BorrowingVault__rebalance_invalidProvider();
   error BorrowingVault__rebalance_invalidFlasher();
   error BorrowingVault__checkFee_excessFee();
+  error BorrowingVault__borrowSlippageProtection();
+  error BorrowingVault__paybackSlippageProtection();
```

Comment:
same comment as for the errors in "BaseVault"

Reply:
fixed as suggested.